src/SUMMARY.md

```diff
@@ -6,18 +6,18 @@
     * [Install Golang](get-started/install-golang.md)
     * [Get the code, build and run](get-started/build-tidb-from-source.md)
     * [Setup an IDE](get-started/setup-an-ide.md)
-    * [Write and run unit tests](get-started/write-and-run-unit-tests.md)
-    * [Commit code and submit a pull request](get-started/commit-code-and-submit-a-pull-request.md)
     * [Debug and profile](get-started/debug-and-profile.md)
+    * [Write and run unit tests](get-started/write-and-run-unit-tests.md)
 
```

Comment:
in the beginning, this section is meant to cover a full lifecycle of download code, write small changes and tests, submit a pull request.

Reply:
Do you mean "Commit code and submit a pull request" lives in "Get Started" Chapter? It's fine to me.